### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by AniLeo**
More IDs for cellSysmodule: libad_core, libad_async, libad_billboard_util (seems they're related to ads / libmedi)
Although found out these 3 modules belong to those 3 IDs, I'm not sure if the order they're matched is correct. 
Should not be important for now though, but should probably be reviewed later.

Related thread that mentions the existence of ads in a game's loading screen back in 2010
http://www.wipeoutzone.com/forum/archive/index.php/t-6996-p-2.html

### CODE REVIEW

**Comment by Zangetsu38** [comment]
hmm fail rebase ? ^^

**Comment by AniLeo** [comment]
I'm still rebasing

**Comment by Zangetsu38** [comment]
ah ok good now :D

File: rpcs3/Emu/Cell/lv2/sys_ss.h [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "Emu/Memory/Memory.h"
+#include "Emu/Cell/ErrorCodes.h"
+
+struct CellSsOpenPSID
+{
```
Comment (Nekotekina) [inline rpcs3/Emu/Cell/lv2/sys_ss.h]: This is be_t<u64>

**Pull request merged**